### DISCUSSION TITLE
Cannot disable full text search

### DISCUSSION THREAD

**Discussion by tom-aptly** [AntlerVC/firetable#1355] [Support / Q&A] [answered]
I am trying to disable full text search so I can delete my project but the UI just hangs at "Disabling tables search..." when I click the button.

**Comment by harinij** [comment, marked as answer]
Hi @tom-aptly Just to confirm, did you delete perms for Rowy / remove Rowy's access to the project directly on GCP before doing this? If so, Rowy wouldn't have access to the project to be able to do so and might be stuck. Also, could you share any console log errors if you see?
